diff --git a/libclay/_clay.pxd b/libclay/_clay.pxd
index c50bfb3..07dd552 100644
--- a/libclay/_clay.pxd
+++ b/libclay/_clay.pxd
@@ -1,6 +1,8 @@
 from libc.stdint cimport uint8_t, int32_t, uintptr_t, uint32_t, uint16_t, int16_t, intptr_t
 
-cdef extern from "libclay/clay.h":
+cdef extern from "clay.h":
+    ctypedef int bool
+    ctypedef void* Clay_Context
 
     uint8_t CLAY__ELEMENT_DEFINITION_LATCH
 
